cli/command/cluster/plugin.go

```diff
@@ -249,28 +291,25 @@ func (p *awsPlugin) Run(c cli.Interface, args []string, env map[string]string) e
 	f := func(r io.Reader, done chan bool) {
 		d := json.NewDecoder(r)
 		for {
-			var m aws.StackOutputList
-			if err := d.Decode(&m); err == io.EOF {
+			err = decodeAwsPluginOutput(c, d)
+			if err == io.EOF {
 				done <- true
 				break
 			} else if err != nil {
-				// If there is an error here, it is because the plugin itself is not returning
-				// json for plugin errors (yet) ... so print the buffer for now
+				done <- true
+				fmt.Println(err)
```

Comment:
Is this `println` required since we have `c.Console().Error(err)` 5 lines below?